pom.xml

```diff
@@ -229,6 +229,19 @@
       <artifactId>hadoop-common</artifactId>
       <version>3.2.2</version>
     </dependency>
+
+    <!-- for pubsub to gcs -->
```

Comment:
Are we sure we need new dependencies.
There are plenty of existing templates with PubSub, GCS and JSON so Ideally we should not be needing it.